code/style.css

```diff
@@ -1,4 +1,310 @@
-html {
-  background: #0025ff;
+header {
+  background:#222222;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+}
+
+/***** LOGO *****/
+.logo img {
+  height: 120px;
+  padding: 12px;
+}
+
+/*** HAMBURGER ***/
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 30px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+}
+
+/*** TOP NAV MENU ***/
+nav {
+  display: none;  /*turned off by default since "Mobile First"*/
+}
+
+nav a {
+  margin-left: 20px;
+}
+
+
+
+/************************************ General formatting ************************************/
+
+html, body {
+  background: #222222;
   color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  text-decoration: none;
+}
+
+
+.big-news-container img, .small-news-container img {
+  box-shadow: 10px 10px #1b1b1b;      /*Box-shadow on all article images (not logo img)*/
+}
+
+h1 {
+  font-size: 32px;
+  line-height: 32px;
+  text-align: left;
+  padding-bottom: 15px;
+  margin-top: 0px;
+  margin-bottom: 5px;
+  padding-top: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 15px;
+}
+
+h2 {
+  font-size: 20px;
+  line-height: 26px;
+  text-align: left;
+  margin-top: 15px;
+  margin-bottom: 5px;
+}
+
+h1, h2 {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+a h1:hover, a h2:hover, nav a:hover, .read-more a:hover {
+  color: rgb(255, 174, 0);
+  text-decoration: underline;
+}
+
+/************************************ THE BIG NEWS ************************************/
+
+.big-news-container {
+  background:#222222;
+  color: #fff;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  flex-direction: column;
+  padding: 10px;
+  margin-bottom: 30px;
+}
+
+.big-news-image {
+  flex-wrap: wrap;
+  width: 100%;
+  background: #222222;
+  padding: 10px;
+  flex-grow: 1;
+  flex-basis: 60%;
+  box-sizing: border-box;
+  display:flex;
+}
+
+.big-news-text .article-preview {   /*styles the article textbox, excluding the headline*/
```

Comment:
Since you do have three different paragraphs, a suggestion would be to give them different class names and hide two of them in mobile and tablet, using `display: none;`
<img width="643" alt="Skärmavbild 2021-02-03 kl  17 54 07" src="https://user-images.githubusercontent.com/51529201/106780964-d14b8d00-6648-11eb-9603-53f91d90428d.png">